specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-01-01/consumption.json

```diff
@@ -1,4547 +1,2930 @@
-          },
-          "UsageDetailsListGroupByTagDate": {
-            "$ref": "./examples/UsageDetailsListGroupByTagDate.json"
-          }
-        },
-        "parameters": [
-          {
-            "$ref": "#/parameters/subscriptionIdParameter"
-          },
-          {
-            "name": "$expand",
-            "description": "May be used to expand the properties/additionalProperties or properties/meterDetails within a list of usage details. By default, these fields are not included when listing usage details.",
-            "in": "query",
-            "required": false,
-            "type": "string"
-          },
-          {
-            "name": "$filter",
-            "description": "May be used to filter usageDetails by properties/usageEnd (Utc time), properties/usageStart (Utc time), properties/resourceGroup, properties/instanceName, properties/instanceId or tags. The filter supports 'eq', 'lt', 'gt', 'le', 'ge', and 'and'. It does not currently support 'ne', 'or', or 'not'. Tag filter is a key value pair string where key and value is separated by a colon (:).",
-            "in": "query",
-            "required": false,
-            "type": "string"
-          },
-          {
-            "name": "$skiptoken",
-            "description": "Skiptoken is only used if a previous operation returned a partial result. If a previous response contains a nextLink element, the value of the nextLink element will include a skiptoken parameter that specifies a starting point to use for subsequent calls.",
-            "in": "query",
-            "required": false,
-            "type": "string"
-          },
-          {
-            "name": "$top",
-            "description": "May be used to limit the number of results to the most recent N usageDetails.",
-            "in": "query",
-            "required": false,
-            "type": "integer",
-            "minimum": 1,
-            "maximum": 1000
-          },
-          {
-            "name": "$apply",
-            "description": "OData apply expression to aggregate usageDetails by tags or (tags and properties/usageStart)",
-            "in": "query",
-            "required": false,
-            "type": "string",
-            "x-ms-parameter-location": "method",
-            "x-ms-parameter-grouping": {
-              "name": "QueryOptions"
-            },
-            "x-ms-client-name": "Apply"
-          },
-          {
-            "$ref": "#/parameters/apiVersionParameter"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK. The request has succeeded.",
-            "schema": {
-              "$ref": "#/definitions/UsageDetailsListResult"
-            }
-          },
-          "default": {
-            "description": "Error response describing why the operation failed.",
-            "schema": {
-              "$ref": "#/definitions/ErrorResponse"
+    "swagger": "2.0",
```

Comment:
Looks like the content got indented 4 spaces instead of 2, so the diff is really hard to see in this comment, as most looks new, could you update indentation?

Reply:
hi @veronicagg , could we change diffsettings to ignore whitespace? I tried it, and I was able to view the specific changes. The indentation is 4 spaces in our other RP (cost.management) and this will introduce consistency. Sorry for the inconvenience.

Reply:
@veronicagg - apparently the space was set to 2 for the other RP as well. I've updated the indentation back to 2.